src/libcollections/vec.rs

```diff
@@ -78,7 +78,6 @@ use core::intrinsics::{arith_offset, assume};
 use core::iter::{FromIterator, FusedIterator, TrustedLen};
 use core::mem;
 #[cfg(not(test))]
-use core::num::Float;
```

Comment:
This is why I think attributes should be on the same line with `use`/`mod`/`extern crate` items.